src/components/Button/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Button.css';
+
+const Button = (props) => {
+	const styles = {
+		backgroundColor: props.color || '#ffa51fe3',
+		width: props.width ? '50%' : '25%'
+	}
+	const { name, handleClick } = props;
+	return(
+	<button onClick={() => props.handleClick(name)} style={styles} className="button">{name}</button>
```

Comment:
We already deconstructed `props` so we don't need to say `props.handleClick` here